app/certificates.go

```diff
@@ -0,0 +1,442 @@
+								}
+								return nil
+							},
+						},
+						&cli.PathFlag{
+							Name:     CaCertificateFileFlagName,
+							Usage:    "The path of the file to store the generated certificate-authority in",
+							Aliases:  []string{"ca-cert"},
+							Required: true,
+						},
+						&cli.PathFlag{
+							Name:     CaPrivateKeyFileFlagName,
+							Usage:    "The path of the file to store the generated certificate-authority's private key in",
+							Aliases:  []string{"ca-key"},
+							Required: true,
+						},
+						&cli.BoolFlag{
+							Name:    "rsa-algorithm",
+							Aliases: []string{"rsa"},
+							Usage:   "Generate the certificate-authority using the RSA algorithm instead of ecdsa (optional)",
+						},
+					},
+					Action: func(ctx *cli.Context) error {
+						duration, err := utils.ParseDuration(ctx.String("duration"))
+						if err != nil {
+							return err
+						}
+						caPem, caPrivKey, err := generateCACertificate(generateCACertificateInput{
+							Organization: ctx.String("organization"),
+							Duration:     duration,
+							RSAAlgorithm: ctx.Bool("rsa-algorithm"),
+						})
+						if err != nil {
+							return fmt.Errorf("failed to generate ca certificate: %s", err)
+						}
+						yes, err := ConfirmPrompt(
+							ctx,
+							fmt.Sprintf("storing certificate authority's private key at %s, do not share this key with anyone. confirm: ",
+								ctx.Path(CaPrivateKeyFileFlagName),
+							),
+						)
+						if err != nil || !yes {
+							return nil
+						}
+						err = ioutil.WriteFile(
+							ctx.Path(CaCertificateFileFlagName),
+							caPem,
+							0600,
+						)
+						if err != nil {
+							return fmt.Errorf("failed to write ca certificate: %s", err)
+
+						}
+						err = ioutil.WriteFile(
+							ctx.Path(CaPrivateKeyFileFlagName),
+							caPrivKey,
+							0600,
+						)
+						if err != nil {
+							return fmt.Errorf("failed to write ca private key: %s", err)
+						}
+
+						return nil
+					},
+				},
+				{
+					Name:    "client-certificate",
+					Usage:   "Generate a client certificate",
+					Aliases: []string{"client"},
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     "organization",
+							Usage:    "The name of the organization",
+							Aliases:  []string{"org"},
+							Required: true,
+						},
+						&cli.StringFlag{
+							Name:  "organization-unit",
+							Usage: "The name of the organization unit (optional)",
+						},
+						&cli.StringFlag{
+							Name:     "duration",
+							Usage:    "The duration of the client certificate, for example: 1y30d (1 year and 30 days)",
+							Aliases:  []string{"d"},
+							Required: true,
+							Action: func(_ *cli.Context, v string) error {
+								if _, err := utils.ParseDuration(v); err != nil {
+									return fmt.Errorf("failed to parse duration: %v", err)
+								}
+								return nil
+							},
+						},
+						&cli.PathFlag{
+							Name:     CaCertificateFileFlagName,
```

Comment:
the name of the command already has the word "ca" in it. Can we remove "ca" from here?